{"uuid": "1f3dc19d-ce33-49f8-8712-415545f4b75d", "children": ["1e3578ad-2a1a-4907-ae7a-b21d4bbfbb4a", "4de02537-8049-402d-b285-a7ba12a168cb"], "befores": [{"name": "test_setup", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.SessionNotCreatedException: Message: session not created: This version of ChromeDriver only supports Chrome version 81\n\n", "trace": "  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\pluggy\\callers.py\", line 187, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 964, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\_pytest\\fixtures.py\", line 788, in call_fixture_func\n    res = next(it)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\conftest.py\", line 6, in test_setup\n    driver = webdriver.Chrome(\"C://Users//Vignesh//PycharmProjects//Selenium//driver//chromedriver.exe\")\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\selenium\\webdriver\\chrome\\webdriver.py\", line 76, in __init__\n    RemoteWebDriver.__init__(\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 157, in __init__\n    self.start_session(capabilities, browser_profile)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 252, in start_session\n    response = self.execute(Command.NEW_SESSION, parameters)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 321, in execute\n    self.error_handler.check_response(response)\n  File \"C:\\Users\\Vignesh\\PycharmProjects\\AutomationFramework1\\venv\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py\", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\n"}, "start": 1590779232804, "stop": 1590779240955}], "start": 1590779232804, "stop": 1590779242253}